src/Behat/Context/FormContext.php

```diff
@@ -62,4 +67,28 @@ public function formElementShouldNotBeRequired($type, $label) {
     }
   }
 
-}
\ No newline at end of file
+  /**
+   * Disable honeypot time limit value.
+   *
+   * To be used when the presence of a honeypot time limit is provoking
+   * false negatives on tests.
+   *
+   * @BeforeScenario @honeypot-disable
+   */
+  public function disableHoneypot() {
+    print('Honeypot: disabling time limit');
+    $this->honeypotTimeLimit = \Drupal::configFactory()->getEditable('honeypot.settings')->get('time_limit');
```

Comment:
Any drupal api call must be placed into either [Drupal8](https://github.com/Metadrop/behat-contexts/blob/master/src/Behat/Cores/Drupal8.php) or [Drupal7](https://github.com/Metadrop/behat-contexts/blob/master/src/Behat/Cores/Drupal7.php) classes. In this case, it should be added into Drupal8 class.

The method would be called like 

```php
   $this->honeypotTimeLimit = $this->getCore()->getHoneypotLimit();
   $this->getCore()->setHoneypotLimit(5);
```

If you do not want to integrate it with drupal7 you can throw a  PendingException, see the example: https://github.com/Metadrop/behat-contexts/blob/master/src/Behat/Cores/Drupal8.php#L61